test/k8sT/DatapathConfiguration.go

```diff
@@ -281,6 +281,7 @@ var _ = Describe("K8sDatapathConfig", func() {
 
 			deployCilium(append(directRoutingOptions,
 				"--set global.endpointRoutes.enabled=true",
+				"--set global.ipv6.enabled=false",
```

Comment:
That's odd, by default ipv6 is not enabled AFAIK.

Reply:
It is enabled in the tests: https://github.com/cilium/cilium/blob/pr/brb/third-host/test/helpers/kubectl.go#L99.